guide/daily-spending-wallet/receiving.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: How receiving works for a daily spending bitcoin wallet.
+nav_order: 2
+parent: Daily spending wallet
+permalink: /guide/daily-spending-wallet/receiving/
+image: /assets/images/guide/daily-spending-wallet/requesting/request-banner-preview(1200x630).jpg
+image_base: /assets/images/guide/daily-spending-wallet/receiving/
+redirect_from:
+ - /guide/payments/receive/
+ - /guide/daily-spending-wallet/funding/
```

Comment:
Should the funding page redirect here? Not really the same content...

Reply:
Redirecting to first use is probably more relevant, what do you think?